src/feedback.tsx

```diff
@@ -91,7 +91,8 @@ const renderFeedbackBody = (
         title,
         description,
         children,
```

Comment:
I'd remove this prop and only allow extra here.

Perform the `children` handling in the components that use this function:
`{icon, title, description, extra: extra ?? children}`

Reply:
done